README.md

```diff
@@ -187,6 +187,12 @@ RNFS.uploadFiles({
        console.log(err);
     });
 ```
+## File Type 
+**All type of Files** ``` 'public.allFiles' or DocumentPickerUtil.allFiles()``` 
+**Only PDF** ``` 'public.pdf' or DocumentPickerUtil.pdf() ```
```

Comment:
You should add blank lines between these, otherwise they'll all be in the same line:

<img width="931" alt="screen shot 2017-09-20 at 17 37 56" src="https://user-images.githubusercontent.com/6205451/30656000-8601c5f6-9e2a-11e7-811d-cb05fb24c09b.png">